packages/fxa-auth-server/config/dev.json

```diff
@@ -427,7 +427,7 @@
     "secondInterval": "2s"
   },
   "mjml": {
-    "enabledEmailAddress": "@testuser.com$",
-    "templates": ["cadReminderFirst"]
+    "enabledEmailAddress": "b@c.com",
```

Comment:
When I set the email back to what Vijay had set in his [PR](https://github.com/mozilla/fxa/commit/3c77c9039b3c6ed22638be9ff0798e170ac08340), CI tests were failing so let's use a new email for now to test the mjml-emails